bedrock/firefox/views.py

```diff
@@ -652,6 +661,29 @@ def get_template_names(self):
             template = ['firefox/whatsnew/index-account.html']
         elif template == ['firefox/whatsnew/whatsnew-fx88-en.html']:
             template = ['firefox/whatsnew/index-account.html']
+        elif template == ['firefox/whatsnew/whatsnew-fx92-en.html']:
+            template = ['firefox/whatsnew/index-account.html']
+
+        return template
+
+
+class WhatsNewEnglishView(WhatsnewView):
+    def get_template_names(self):
+        template = super().get_template_names()
+        if switch('firefox-whatsnew-92-vpn-pricing') and template == ['firefox/whatsnew/whatsnew-fx92-en.html']:
+            template = ['firefox/whatsnew/whatsnew-fx92-vpn-en.html']
+
+        return template
+
+
+class WhatsNewFranceView(WhatsnewView):
+    def get_template_names(self):
+        template = super().get_template_names()
+        locale = l10n_utils.get_locale(self.request)
+        if switch('firefox-whatsnew-92-vpn-pricing') and locale == 'fr':
```

Comment:
Could be slightly simplified by doing:
```python
if switch('firefox-whatsnew-92-vpn-pricing'):
    template = ['firefox/whatsnew/whatsnew-fx92-vpn-fr.html'] if locale == 'fr' else ['firefox/whatsnew/index-account.html']
```

Not a requirement for sure, just FYI mostly.